lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -436,19 +438,25 @@ sub run_backup {
             format => 'HTML',
         );
         return $template->render_to_psgi($request);
-    } elsif ($request->{backup_type} eq 'browser') {
+    }
+    elsif ($request->{backup_type} eq 'browser') {
         my $bak;
         open $bak, '<', $backupfile
             or die "Failed to open temporary backup file $backupfile : $!";
         unlink $backupfile; # remove the file after it gets closed
 
         my $attachment_name = 'ledgersmb-backup-' . time . '.sqlc';
-        return [ 200,
-                 [ 'Content-Type' => $mimetype,
-                   'Content-Disposition' =>
-                        "attachment; filename=\"$attachment_name\"" ],
-                 $bak ];  # return the file-handle
-     } else {
+        return [
+            200,
```

Comment:
coding style question, @ehuelsmann I feel `return [ 200,` is more readable than putting the `200` on the second line, But I'm happy to go with your preference